docs/api.md

```diff
@@ -1129,7 +1129,7 @@ The `elementHandle.dispose` method stops referencing the element handle.
 - returns: <[Promise]<[Serializable]>> Promise which resolves to function return value
 
 If the function, passed to the `elementHandle.evaluate`, returns a [Promise], then `elementHandle.evaluate` would wait for the promise to resolve and return it's value.
-The function will be passed in the element ifself as a first argument.
+The element ifself will be passed in the function as a first argument.
```

Comment:
Yes, this is still true: the element itself is passed as a first argument, followed by all the arguments. (note: this is `elementHandle.evaluate`, not `page.evaluate`)